### DISCUSSION TITLE
TUM student project to contribute to this repo

### DISCUSSION THREAD

**Discussion by felix-ht** [dartclub/turf_dart#103] [Ideas]
I wonder if there would be interest in the following:

My company would setup an IDP (similar to an internship for teams) project for TUM informatics master students. We would provide support as well as the connection to TUM as well (initial) code reviews, as well as selecting the team.

They would contribute to this project and implement some of missing functions.

To give you some more context:
I am one of the main contributors to https://github.com/flutter-mapbox-gl/maps and moving this project forward would be great for the whole dart ecosystem. My company heavily relies on the ability to interact with geojson even without any server connection - so this is just what we need.

It would certainly take a couple of month until a potential kickoff (next semester?).

**Comment by lukas-h** [comment]
Hi Felix,

That sounds really amazing!!!

I already know you from GitHub. 😉 I follow Flutter-Mapbox’s development closely.

My company is atm getting started with some contributions to Flutter-Mapbox as well!
And we rely a lot on Turf and Mapbox as a critical part of some of our projects.

This week, we had a technical discussion about the library with @tobrun, amongst others, where we discussed how we want to ramp up the development efforts of this library.

The programme with the TUM students would help this OSS project tremendously and is extremely well aligned with our goals!

Let‘s have a meeting to get to know each other and discuss some details!

My contact details: https://himsel.me
I‘m from Nuremberg, btw.

**Really** looking forward to a meeting,
Lukas

CC @armantorkzaban

**Comment by lukas-h** [comment]
Hi @felix-ht,

Let‘s talk soon. I see a lot of potential in this cooperation!

**Reply by felix-ht** [reply]
Been pretty recently - and on holiday next week - will come back to you after that!

**Reply by armantorkzaban** [reply]
Enjoy the holiday. Looking forward to our first meeting on the topic.
